File: DSGTestNet/Frms/FrmInfo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DSGTestNet.Frms
{
    public partial class FrmInfo : Form
    {
        public FrmInfo()
        {
            InitializeComponent();
            IniListInput += FrmInfo_IniListInput;
             
        }

        /// <summary>
        /// 初始化排产队列信息
        /// </summary>
        /// <param name="sender"></param>
        /// <param name="e"></param>
        private void FrmInfo_IniListInput(object sender, EventArgs e)
        {
            txtInfo.Text = "asdfasdfasd";
        }

        /// <summary>
        /// 鼠标按下左键是的坐标点
        /// </summary>
        Point _mousePoint = new Point();

        public event EventHandler IniListInput;

        private void FrmInfo_MouseDown(object sender, MouseEventArgs e)
        {
            if (e.Button == MouseButtons.Left)
                _mousePoint = e.Location;
        }

        private void FrmInfo_MouseMove(object sender, MouseEventArgs e)
        {
            if (e.Button == MouseButtons.Left)
            {
                int x = e.X - _mousePoint.X;
                int y = e.Y - _mousePoint.Y;
                Location = Point.Add(Location, new Size(x, y));
            }
        }

        
    }
}

File: 神龙三厂胎压设备搬迁项目VS/frmInfo.vb
Option Strict Off
Option Explicit On
Friend Class frmInfo
	Inherits System.Windows.Forms.Form

    '鼠标按下左键是的坐标点
    Private mousePoint As Point

    Private Sub frmInfo_MouseDown(sender As Object, e As MouseEventArgs) Handles MyBase.MouseDown
        If e.Button = MouseButtons.Left Then
            mousePoint = e.Location
        End If
    End Sub

    Private Sub frmInfo_MouseMove(sender As Object, e As MouseEventArgs) Handles MyBase.MouseMove
        If e.Button = MouseButtons.Left Then
            Dim x As Integer
            Dim y As Integer
            x = e.X - mousePoint.X
            y = e.Y - mousePoint.Y
            Location = Point.Add(Location, New Size(x, y))
        End If
    End Sub
End Class